### ISSUE TITLE
Question: Is it possible to disable setPointerCapture() on drag events in version >= 9 ? 

### ISSUE THREAD

**Issue by tokarchyn** [pmndrs/use-gesture#264]
I'm writing library for a diagramming and my use case looks like this:
- There are multiple divs, that do not change their position. They have attached onDrag[Start, Stop] and onPointer[Enter, Leave] handlers
- User start dragging on div number '1', from the event's position I start drawing the line to the mouse position
- User hover over the div number '2' (dragging is still active), I validate these two divs if they can be connected, and change their colors depending on this.
- If divs can be connected and user release mouse button or touch, I draw the line between those divs

When I use react-use-gesture 8.0.1, everything worked ok, with a little hack on touch devices - in the onDragStart handler I released pointer capture like this (removing this code will not solve my problem):
```
onDragStart: ({ event }) => {
  // Important! Otherwise on touch display onPointerEnter will not work!
  event.target.releasePointerCapture(event.pointerId); 
  ...
```
Unfortunately, after upgrading library to version 9.0.3,  **onPointerEnter event is not raised during dragging** (dragging still works) :(

How it looks now:
<img src="https://user-images.githubusercontent.com/20689583/107997767-e4187700-6fe3-11eb-9074-f7992a05d0ad.gif" width="400">
How it should looks:
<img src="https://user-images.githubusercontent.com/20689583/107997860-22159b00-6fe4-11eb-8aa8-b25a466c07cd.gif" width="400">

**Comment by dbismut** [comment]
Fair enough, the use case makes sense :) I'll think about what could be the best API for this tomorrow.

**Comment by dbismut** [comment]
A bit harder than I thought. For this to work I need to revert back to attaching events to the window instead of setPointerCapture.

**Comment by dbismut** [comment]
Hi @tokarchyn sorry for the long wait but here is a sandbox showing progress on v10.

https://codesandbox.io/s/gesture-drag-9pt0m

If you leave the `capture` checkbox set to `true`, you'll notice that the blue square hover css state doesn't get trigger when dragging the red square.  If you untick the `capture` checkbox, the css state will trigger while dragging (you need to move your mouse fast enough so that the square trails).

Note that this doesn't seem to work in Safari but that's (just another) perk of this browser unrelated to this lib (pretty sure js events would trigger so that should solve your usecase).


https://user-images.githubusercontent.com/5003380/116257784-a28b1180-a774-11eb-897a-5b277c054a18.mov

**Comment by dbismut** [comment]
Closing as fixed in v10 #300. Please let me know if the issue persists somehow.
